packages/build/src/status/validations.ts

```diff
@@ -0,0 +1,19 @@
+import { SecretScanResult } from '../plugins_core/secrets_scanning/utils.js'
+
+// Reports any validations completed on the deploy to the API
+export const reportValidations = async function ({
+  api,
+  secretScanResult,
+  deployId,
+}: {
+  api: unknown
+  secretScanResult: SecretScanResult
+  deployId: string
+}) {
+  try {
+    // @ts-expect-error API type is not defined
+    api.updateDeployValidations({ deploy_id: deployId, body: { secrets_scan: secretScanResult } })
+  } catch {
+    // Noop
```

Comment:
This completely silences any errors/exceptions raised during this call, I'm just making sure that's what we want to do. It seems like we might at least want to log something somewhere.

Are there any known exceptions with this call?

Reply:
if it should not be visible but we should still be able to look up maybe `systemLog` could report some error here?

Reply:
I've added a basic `systemLog` for now (thanks for the suggestion!) so we can have some visibility into failures